internal/nexodus/policy_windows.go

```diff
@@ -11,3 +13,13 @@ func (nx *Nexodus) processSecurityGroupRules() error {
 func (nx *Nexodus) nfNetworkRouterSetup() error {
 	return nil
 }
+
+// runNftCmd for windows build purposes
+func runNftCmd(logger *zap.SugaredLogger, cmd []string) (string, error) {
+	return "", nil
+}
+
+// nfTableDrop for windows build purposes
+func (nx *Nexodus) nfTableDrop(table string) error {
+	return nil
+}
```

Comment:
how about returning errors in case they're ever called by accident for some reason